lib/ddtrace/contrib/action_pack/action_controller/instrumentation.rb

```diff
@@ -130,7 +130,6 @@ def process_action(*args)
                 result
               # rubocop:disable Lint/RescueException
               rescue Exception => e
-                payload[:exception] = [e.class.name, e.message]
                 payload[:exception_object] = e
                 raise e
```

Comment:
Looks like we were trying to emulate ActiveSupport::Notifications internals: https://github.com/rails/rails/blob/93b7bd5ce23a96548bf93667e416037dee40d90d/activesupport/lib/active_support/notifications/instrumenter.rb#L25-L28

But the `payload` object is only used by us, never returned to the Rails framework.